tests/integration_tests/test_timeseries_feature.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+import torch
+
+from ludwig.features.timeseries_feature import TimeseriesInputFeature
+from tests.integration_tests.utils import timeseries_feature
+
+BATCH_SIZE = 2
+SEQ_SIZE = 10
+DEFAULT_FC_SIZE = 256
+
+
+@pytest.mark.parametrize(
+    'enc_encoder',
+    [
+        'stacked_cnn', 'parallel_cnn', 'stacked_parallel_cnn', 'rnn', 'cnnrnn',
+        'passthrough'
+    ]
+)
+def test_timeseries_feature(enc_encoder):
+    # synthetic time series tensor
+    timeseries_tensor = torch.randn([BATCH_SIZE, SEQ_SIZE],
+                                    dtype=torch.float32)
+
+    # generate feature config
+    timeseries_feature_config = timeseries_feature(
+        encoder=enc_encoder,
+        max_len=SEQ_SIZE,
+        # simulated parameters determined by pre-processing
+        max_sequence_length=SEQ_SIZE,
+        embedding_size=1,
+        should_embed=False
```

Comment:
These 2 paramters shouldn't be needed, as they should be the defaults

Reply:
@w4nderlust  
As I understand the code, the two parameters are not "default" per se but are set when the feature's configuration is [updated with results of creating the feature's metadata](https://github.com/jimthompson5802/ludwig/blob/f497e111874fddea10822331762f0cd24d591c3b/ludwig/features/timeseries_feature.py#L178-L187l).   The only [default values set are these](https://github.com/jimthompson5802/ludwig/blob/f497e111874fddea10822331762f0cd24d591c3b/ludwig/features/timeseries_feature.py#L190-L197).

I can implement the comment by moving those two parameters from `update_config_with_metadata()` to `populate_defaults()` method and then calling `TimeSeriesInputFeature.populate_defaults()` in the unit test.  So the unit test code will look something like this.
```
    # generate feature config
    timeseries_feature_config = timeseries_feature(
        encoder=enc_encoder,
        max_len=SEQ_SIZE,
        fc_layers=[{'fc_size': DEFAULT_FC_SIZE}],
        # simulated parameters determined by pre-processing
        max_sequence_length=SEQ_SIZE,
    )

    # instantiate input feature object
    TimeSeriesInputFeature.populate_defaults(timeseries_feature_config)
    timeseries_input_feature = TimeseriesInputFeature(timeseries_feature_config)
```

Does this sound reasonable?

Reply:
You have a good point. The issue though is that those two parameters are parameters for the encoders really, and no other parameter would work. So probably a better idea would be not to set them at all, and just pass them as fixed values when initializing the encoder. Something like
```
self.encoder = encoder_cls(embedding_size=1,  should_embed=False, **other_params)
```

Reply:
OK...made requested change and pushed up.  Unit test pass with the changes
```
 ============================= test session starts ==============================
collecting ... collected 6 items

../../tests/integration_tests/test_timeseries_feature.py::test_timeseries_feature[stacked_cnn] 
../../tests/integration_tests/test_timeseries_feature.py::test_timeseries_feature[parallel_cnn] 
../../tests/integration_tests/test_timeseries_feature.py::test_timeseries_feature[stacked_parallel_cnn] 
../../tests/integration_tests/test_timeseries_feature.py::test_timeseries_feature[rnn] 
../../tests/integration_tests/test_timeseries_feature.py::test_timeseries_feature[cnnrnn] 
../../tests/integration_tests/test_timeseries_feature.py::test_timeseries_feature[passthrough] 

======================== 6 passed, 3 warnings in 0.21s =========================
```